src/main_controller.ts

```diff
@@ -129,6 +130,9 @@ export class MainController implements vscode.Disposable {
         if (settings.clean) {
             args.push("--clean");
         }
+        if (settings.neovimCmd) {
+            args.push("--cmd", settings.neovimCmd);
```

Comment:
I don't know the answer to this, but what happens if `neovimCmd` contains other flags, like "--clean"... will it be properly escaped?

Reply:
The argument after `--cmd` is always parsed and executed as vimL.

Reply:
So if someone wants more control (to pass multiple `-c`, `--cmd`, etc. args), we'll have to add another option. 

To avoid that, perhaps this option should be `nvimArgs` array. Then people can have settings like:

```
"nvimArgs" = [
  "--cmd", "set nowrap",
  "-c", "edit foo",
  ...
]
```

Reply:
I had originally considered this, but command-line options are already used to implement other settings. For example, if  `vscode-neovim.neovimClean` is set to true along and `nvimArgs = [ "--cmd" ]` then the final neovim command would be `nvim --cmd --clean` which would result in an error. 

The solution would be to append `nvimArgs` at the end after everything else if set. Perhaps we could even name it `nvimExtraArgs`.

Reply:
> The solution would be to append `nvimArgs` at the end after everything else is set.

Yes, of course. But even just one command can break stuff, depending on what is sent. 

The setting description can mention what the default args are.